src/main/java/io/github/dsheirer/controller/channel/ChannelProcessingManager.java

```diff
@@ -30,15 +32,13 @@
 import io.github.dsheirer.controller.channel.event.ChannelStartProcessingRequest;
 import io.github.dsheirer.controller.channel.event.PreloadDataContent;
 import io.github.dsheirer.controller.channel.map.ChannelMapModel;
-import io.github.dsheirer.identifier.Form;
-import io.github.dsheirer.identifier.Identifier;
-import io.github.dsheirer.identifier.IdentifierClass;
-import io.github.dsheirer.identifier.IdentifierUpdateNotification;
+import io.github.dsheirer.identifier.*;
```

Comment:
Try to avoid using wildcard imports. It increases the odds of a namespace collision breaking the build randomly in the future.

Reply:
Yeah that was done silently by my editor